docs/REST API/rest-api.md

```diff
@@ -8723,6 +8723,73 @@ Gets the result of a script that was executed.
 
 > Note: `created_at` is the creation timestamp of the script execution request.
 
+
+### Batch-run script
+
+Run a script on multiple hosts.
+
+The script will be added to each host's list of upcoming activities.
+
+`POST /api/v1/fleet/scripts/run/batch`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                                                                    |
+| ----            | ------- | ---- | --------------------------------------------                                                   |
+| script_id       | integer | body | **Required**. The ID of the existing saved script to run. |
+| host_ids        | array   | body |  List of host IDs.  Required if `filters` not specified. Only one of `host_ids` or `filters` may be included in the request.   |                                            |
+| filters | object  | body | See [filters](#filters3). Required if `host_ids` not specified. Only one of `host_ids` or `filters` may be included in the request.   |
+
+
+##### Filters
```

Comment:
Do we want to omit each of these fields when empty  i.e., if there's no search query, will omit `filters.query` altogether, instead of sending `filters.query: ""`

Reply:
That's fine, it'll be interpreted the same way on the back end but cleaner to leave them out IMO.

Reply:
Agreed